src/core/storage/utils/state-helpers.ts

```diff
@@ -146,13 +85,129 @@ export async function readStateFromDisk(context: ExtensionContext) {
 		context.secrets.get("ollamaApiKey") as Promise<string | undefined>,
 		context.secrets.get("vercelAiGatewayApiKey") as Promise<string | undefined>,
 		context.secrets.get("difyApiKey") as Promise<string | undefined>,
+		context.secrets.get("authNonce") as Promise<string | undefined>,
 	])
 
+	return {
+		authNonce,
+		apiKey,
+		openRouterApiKey,
+		clineAccountId,
+		huggingFaceApiKey,
+		huaweiCloudMaasApiKey,
+		basetenApiKey,
+		zaiApiKey,
+		ollamaApiKey,
+		vercelAiGatewayApiKey,
+		difyApiKey,
+		sapAiCoreClientId,
+		sapAiCoreClientSecret,
+		xaiApiKey,
+		sambanovaApiKey,
+		cerebrasApiKey,
+		groqApiKey,
+		moonshotApiKey,
+		nebiusApiKey,
+		asksageApiKey,
+		fireworksApiKey,
+		liteLlmApiKey,
+		doubaoApiKey,
+		mistralApiKey,
+		openAiNativeApiKey,
+		deepSeekApiKey,
+		requestyApiKey,
+		togetherApiKey,
+		qwenApiKey,
+		geminiApiKey,
+		openAiApiKey,
+		awsBedrockApiKey,
+		awsAccessKey,
+		awsSecretKey,
+		awsSessionToken,
+	}
```

Comment:
does this verify we are not removing / missing any key that's current working / being used?